src/index.js

```diff
@@ -22,8 +22,12 @@ var Script2 = {
         if (!this.src) {
           Script2.p = Script2.p.then(() => {
             var s = document.createElement('script')
+            var h = this.$el.innerHTML;
+            h = h.replace(/&lt;/gi, '<');
+            h = h.replace(/&gt;/gi, '>');
+            h = h.replace(/&amp;/gi, '&');
```

Comment:
These `h = h`'s can be shortened to `var h = this.$el.innerHTML.replace( .. ).replace( .. ). replace( .. )`.

Also, don't forget to remove semi-colons to ensure [`standard`](https://github.com/standard/standard) linter passes.